frontend/javascripts/oxalis/view/right-menu/comment_tab/comment_tab_view.js

```diff
@@ -145,6 +157,28 @@ class CommentTabView extends React.PureComponent<PropsWithSkeleton, CommentTabSt
     );
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if (nextState !== this.state) {
+      return true;
+    }
+
+    if (this.props.skeletonTracing.activeNodeId !== nextProps.skeletonTracing.activeNodeId) {
+      return true;
+    }
+    if (this.props.skeletonTracing.activeTreeId !== nextProps.skeletonTracing.activeTreeId) {
+      return true;
+    }
+
+    const updateActions = Array.from(
+      cachedDiffTrees(this.props.skeletonTracing, nextProps.skeletonTracing),
+    );
+
+    const relevantUpdateActions = updateActions.filter(ua =>
+      RELEVANT_ACTIONS_FOR_COMMENTS.includes(ua.name),
+    );
```

Comment:
To avoid that the comment tab does not update when NMLs have imported the following works but the code does not look very good :/ Maybe you can think of a beeter way to integrate this behaviour into the code :thinking: 
```suggestion
    const relevantUpdateActions = updateActions.filter(
      ua =>
        RELEVANT_ACTIONS_FOR_COMMENTS.includes(ua.name) ||
        (ua.name === "createTree" && ua.value.comments.length > 0),
    );
```

Reply:
Thanks again for the testing and also for the provided fix! I like your suggestion and think that it's quite fitting :)